modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -223,17 +222,8 @@ class Create_Timepoint extends \NDB_Form
                 $num_sites          = count($user_list_of_sites);
                 if ($num_sites > 1) {
                     $this->tpl_data['pscLabelAdded'] = true;
-                    $psc_labelOptions = [null => ''];
-                    foreach ($user_list_of_sites as $key => $siteID) {
-                        $center = $DB->pselectRow(
-                            "SELECT CenterID as ID, Name FROM psc 
-                        WHERE CenterID =:cid",
-                            ['cid' => $siteID]
-                        );
-                        if (!is_null($center)) {
-                            $psc_labelOptions[$siteID] = $center['Name'];
-                        }
-                    }
+                    $psc_labelOptions = [null => ''] +
```

Comment:
I don't think you can use `+` to glue arrays together.

Reply:
why not?

Reply:
as far as I know PHP doesn't work like that.

```php
php > $arr1 = [1,2,3];
php > $arr2 = [4,5,6];
php > print_r($arr1 + $arr2);
Array
(
    [0] => 1
    [1] => 2
    [2] => 3
)
```

Reply:
Oh, this is different. You are trying one dimensional array.